src/pages/demo/partials/benefitHubsLinks.tsx

```diff
@@ -19,24 +25,21 @@ const DemoPage = ({ nodes }) => {
 }
 export default DemoPage
 
-export async function getStaticProps(context) {
+export async function getStaticProps(
+  context: GetStaticPropsContext
+): Promise<GetStaticPropsResult<BenefitsHubLinksProps>> {
   const params = new DrupalJsonApiParams()
   params
     .addFilter('status', '1')
     .addFilter('field_related_benefit_hubs', null, 'IS NOT NULL')
```

Comment:
Instead of 'IS NOT NULL' you can use Operators.isNotNull

Reply:
perfect, nice find we should probably go back and update the some of the other original components with that fix